app/controllers/books_controller.rb

```diff
@@ -1,8 +1,14 @@
 class BooksController < ApplicationController
-  def show
-    book = BooksService.new.find_book(params[:isbn])
-    return render json: { message: 'Book not found' }, status: :not_found if book.blank?
+  include Wor::Paginate
```

Comment:
you can move this to ApplicationController or create a new ApiController and inherit all API controller fo this